plugins/source/aws/resources/services/elasticache/gen.hcl

```diff
@@ -123,4 +123,575 @@ resource "aws" "elasticache" "clusters" {
+    description = "Pending modified auth token status"
+  }
+
+  column "pending_modified_values_automatic_failover_status" {
+    rename = "pending_automatic_failover_status"
+    description = "pending autmatic failover for this redis replication group"
+  }
+
+  # Actually skipping a relation.
+  column "pending_modified_values_log_delivery_configurations" {
+    skip = true
+  }
+
+  column "pending_modified_values_primary_cluster_id" {
+    rename = "pending_primaty_cluster_id"
+  }
+
+  column "pending_modified_values_resharding_slot_migration_progress_percentage" {
+    rename = "pending_resharding_slot_migration_progress_percentage"
+  }
+
+  column "pending_modified_values_user_groups_user_group_ids_to_add" {
+    rename = "pending_user_group_ids_to_add"
+  }
+
+  column "pending_modified_values_user_groups_user_group_ids_to_remove" {
+    rename = "pending_user_group_ids_to_remove"
+  }
+
+  relation "aws" "elasticache" "log_delivery_configurations" {
+    column "destination_details_cloud_watch_logs_details_log_group" {
+      rename = "cloudwatch_destination_log_group"
+      description = "The log group of the CloudWatch Logs destination"
+    }
+
+    column "destination_details_kinesis_firehose_details_delivery_stream" {
+      rename = "kinesis_firehose_destination_delivery_stream"
+      description = "The Kinesis Data Firehose delivery stream of the Kinesis Data Firehose destination"
+    }
+  }
+
+  relation "aws" "elasticache" "node_groups" {
+    relation "aws" "elasticache" "node_group_members" {
+      rename = "members"
+    }
+  }
+}
+
+resource "aws" "elasticache" "reserved_cache_nodes" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.ReservedCacheNode"
+
+  description = "Reserved Elasticache Cache Nodes"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = [
+      "reservation_arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+}
+
+resource "aws" "elasticache" "reserved_cache_nodes_offerings" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.ReservedCacheNodesOffering"
```

Comment:
This table also probably deserves a description

Reply:
Same - autofilled by cq-gen.